src/nodes/network-requester-setup.md

```diff
@@ -223,6 +173,20 @@ curl -d '{"since":"2022-07-26T12:46:00.000000+00:00", "until":"2022-07-26T12:57:
 ```
 ~~~
 
+## Network Requester Whitelist
+
+The network requester is not an open proxy. It uses a file called `allowed.list` (in `~/.nym/service-providers/network-requester/`) as a whitelist for outbound requests. 
+
+Any request to a URL which is not on this list will be blocked.
+
+On startup, if this file is not present, the requester will grab the default whitelist from [here](https://nymtech.net/.wellknown/network-requester/standard-allowed-list.txt) automatically. 
+
+> This default whitelist is useful for knowing that the majority of network requesters are able to support certain apps 'out of the box'. 
+
+> Operators of a network requester are of course free to edit this file and add the URLs of services they wish to support to it. You can find instructions below on adding your own URLs or IPs to this list. 
+
+The `nym-network-requester` binary listens and sends responses via its embedded `nym-client`, which is how it connects to the mixnet. 
```

Comment:
the client is no longer seperate from the network requester binary - make this clear